src/server.rs

```diff
@@ -283,10 +285,78 @@ async fn upload(
     Ok(HttpResponse::Ok().body(urls.join("")))
 }
 
+/// File entry item for list endpoint.
+#[derive(Serialize, Deserialize)]
+pub struct ListItem {
+    /// Uploaded file name.
+    pub file_name: String,
+    /// Size of the file in bytes.
+    pub file_size: u64,
+    /// ISO8601 formatted date-time string of the expiration timestamp if one exists for this file.
+    pub expires_at: Option<String>,
```

Comment:
Sorry for my question, but isn't a noun better? e.g. `expiry` or `expiration`

I am not sure about that one, but I believe I have seen more nouns as keys than verbs.

Reply:
I usually use `_at` in my timestamps so it can be known that the field is a timestamp just by reading the name, but that's mostly a holdover from my days as a Rails programmer (where it is a convention). That reasoning makes less sense in a typed language, though. I personally am not dogmatic about using nouns as property fields, I tend to use whatever I think makes the meaning most clear. I'm fine changing it though, especially if there is already a convention being used to follow.

Reply:
No need to change it. It was just something that came to mind.